src/api.cpp

```diff
@@ -112,7 +112,7 @@ namespace Rcpp {
         case BCODESXP:  	return "BCODESXP";
         case EXTPTRSXP: 	return "EXTPTRSXP";
         case WEAKREFSXP:    return "WEAKREFSXP";
-#if R_Version >= R_Version(4,4,0)					// replaces S4SXP in R 4.4.0
+#if R_VERSION >= R_Version(4,4,0)					// replaces S4SXP in R 4.4.0
```

Comment:
This one seemed a bit strange; I'm not sure if this really worked before (or, if it did, _why_ it did)?

Reply:
Yeah that looks like you spotted an 'ooops'. Thank you!

And thanks for the update.  It look good.  Will have to run another rev.dep run but we should be good by end of week. Fingers crossed!